### DISCUSSION TITLE
Panel Configuration Export/Import Feature

### DISCUSSION THREAD

**Discussion by PsychoZander** [pelican-dev/panel#213] [Ideas]
Create a tool to allow simple Panel configuration exports and imports, for use as a super basic backup or migration.

**Comment by Poseidon281** [comment]
You mean like being able to backup the database from the Panel?

**Reply by PsychoZander** [reply]
I guess it would be, yes. I know it's quite a complex system to reduce to something as simple as exporting a single file, however I do think this has some real value. Whilst the Dockerisation of the panel, as planned, will allow for easier migrations it's still a relatively complex procedure. I would not expect Nodes or Servers to be included, or at least to maybe include a 1 liner generated by the import wizard to run on the node to transfer it to the new panel.

**Comment by ItsNeil17** [comment]
This would actually be a good feature, just not one to add/worry about before a stable version is released.

While yes, top level panel exports would be good for containerization, not many people will be exporting/importing their panels.
